debezium-core/src/main/java/io/debezium/transforms/Filter.java

```diff
@@ -178,31 +120,32 @@ public static NullHandling parse(String value, String defaultValue) {
     private Engine engine;
     private NullHandling nullHandling;
 
-    public Filter() {
-    }
-
     @Override
     public void configure(Map<String, ?> configs) {
         final Configuration config = Configuration.from(configs);
-        final ExpressionLanguage language = ExpressionLanguage.parse(config.getString(LANGUAGE));
-        final String expression = config.getString(EXPRESSION);
-        if (language == null || expression == null) {
-            throw new DebeziumException(
-                    "Options '" + LANGUAGE + "' and '" + EXPRESSION + "' must be present. Language must be one of (" + Arrays.toString(ExpressionLanguage.values())
-                            + ")");
-        }
-        LOGGER.info("Using language '{}' to evaluate expression '{}'", language.getValue(), expression);
+        final String expression = config.getStringNonEmpty(EXPRESSION);
+        String language = config.getStringNonEmpty(LANGUAGE);
+
+        LOGGER.info("Using language '{}' to evaluate expression '{}'", language, expression);
 
         nullHandling = NullHandling.parse(config.getString(NULL_HANDLING));
 
-        try {
-            engine = language.getEngine().newInstance();
+        if (!language.startsWith(JAVAX_SCRIPT_ENGINE_PREFIX)) {
```

Comment:
@gunnarmorling Maybe add a comment that this is only temporary restriction?

Reply:
Pushed an update for that.